### PR TITLE
Feature/OB-5

### PR DESCRIPTION
**Pull request by csabbee**

### CODE REVIEW

File: app/components/editable-list-element.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        const newUser = Object.assign({}, user, { data: userData })
+        updateUser({ user: newUser, id: listElement.id });
+    }
+
+    return (
+        <li>
+            <label>
+                Title:
+                <input value={listElement.title} onChange={updateEntry} name="title"/>
```
Comment (ppseprus) [inline app/components/editable-list-element.jsx]: No space before "/>".

File: app/components/editable-list-element.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            </label>
+            <label>
+                Url:
+                <input value={listElement.url} onChange={updateEntry} name="url" />
```
Comment (ppseprus) [inline app/components/editable-list-element.jsx:44]: [Or] Extra space before "/>".

File: app/components/editable-list-element.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        }
+    };
+};
+
+
+const EditableListElement = ({ saveEntry, user, updateUser, listElement }) => {
+    function updateEntry(event) {
+        const property = event.target.name;
+        const userData = user.data.map(element => {
+            if (element === listElement) {
```
Comment (ppseprus) [inline app/components/editable-list-element.jsx:23]: Should this work? Shouln't you compare IDs instead? (...or something)

**Review by ppseprus — APPROVED** [review]

File: app/components/editable-list-element.jsx (hunk 3 above)
Comment (csabbee) [reply app/components/editable-list-element.jsx:23]: Yes this is working since we are having one state and we are getting the data from that.

Comment (ppseprus) [reply app/components/editable-list-element.jsx:23]: Cool! : )

**Pull request merged**